### PR TITLE
Issue 12 update layout include header footer body

### PR DESCRIPTION
**Pull request by Wolfinbarger**
Current layout with dummy card for games.

image

### CODE REVIEW

File: src/app/app/components/avatar/avatar.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+div {
+  display: flex;
+  gap: 4%;
+  align-items: center;
+  justify-content: center;
+  // margin-top: auto;
```
Comment (FreakyNobleGas) [inline src/app/app/components/avatar/avatar.scss:6]:
```suggestion
```

File: src/app/components/avatar/avatar.scss
Comment (FreakyNobleGas) [inline src/app/components/avatar/avatar.scss:1]: Are these duplicate files?

File: src/app/components/avatar/avatar.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+div {
+  display: flex;
+  gap: 4%;
+  align-items: center;
+  justify-content: center;
+  // margin-top: auto;
```
Comment (FreakyNobleGas) [inline src/app/components/avatar/avatar.scss:6]:
```suggestion
```

**Review by FreakyNobleGas — COMMENTED** [review]
This is awesome. I had a couple questions.

File: src/app/app/components/Games.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+async function getGames() {
+  const url = "http://127.0.0.1:8000/api/games/";
+  const res = await fetch(url);
+
+  return res.json();
+}
+
+export default async function Games() {
+  const games = await getGames();
+  const alphaGames = games.data.sort();
```
[the remark this answers is not in the document]
Comment (Wolfinbarger) [reply src/app/app/components/Games.jsx:10]: That is not being used nor is it used in issue 10.

**Review by FreakyNobleGas — APPROVED** [review]
Nice! 🚀

**Pull request merged**